app/Plugin/LinOTPAuth/Controller/Component/Auth/LinOTPAuthenticate.php

```diff
@@ -33,21 +110,53 @@ public function getUser(CakeRequest $request)
         $userFields = $request->data['User'];
         $email = $userFields['email'];
         $password = $userFields['password'];
+        $otp = $userFields['otp'];
+
         CakeLog::debug("getUser email: ${email}");
 
-        $linotp = new LinOTP(
-            Configure::read("LinOTPAuth.baseUrl"),
-            Configure::read("LinOTPAuth.realm")
-        );
+        $linOTP_baseUrl = rtrim(Configure::read("LinOTPAuth.baseUrl"), "/");
```

Comment:
Fallback value if `LinOTPAuth.baseUrl` is unset?

Reply:
Don't think a fallback is useful - but a check before that, outright breaking the execution if it's not configured would be.